server/tests/test_data/nodejs_query/timeline_all/screenshot.json

```diff
@@ -18,7 +18,8 @@
         "date": "2019",
         "value": 105819
       },
-      "chartUrl": ""
+      "chartUrl": "",
+      "dcUrl": "https://datacommons.org/explore#q=family%20earnings%20in%20north%20dakota"
```

Comment:
nit: we should have a TODO to switch this to bard.datacommons.org (in a couple of days)

Reply:
will include this in followup PR that will also fix the ranking bug